### ISSUE TITLE
Allow Using PneumaticCraft and Industrial Foregoing Plastic interchangeably

### ISSUE THREAD

**Issue by BlueNexus** [NillerMedDild/Enigmatica6#1991]
**Modpack version**
4.8.0
**Describe the bug**
Both items are called plastic, so it makes sense that you'd be able to use them interchangably.

**Comment by NillerMedDild** [comment]
Thanks for the report, we'll get that added.

**Comment by KrAzYGEEK32** [comment]
i personally feel like this would break PNC progression. plastic in PNC is a late game tier item and it takes a lot of effort to make it, automating it is another big story. where as IF plastic is entry tier and easily automatable. if these are interchangable, you would be bypassing all of PNC oil processing and a major part of its gameplay and challenges.
TL;DR : don't do it :).

**Comment by sciguyryan** [comment]
> 
> 
> i personally feel like this would break PNC progression. plastic in PNC is a late game tier item and it takes a lot of effort to make it, automating it is another big story. where as IF plastic is entry tier and easily automatable. if these are interchangable, you would be bypassing all of PNC oil processing and a major part of its gameplay and challenges.
> TL;DR : don't do it :).

Another option would be to make the IF recipes use the PNC plastic, though that would have the impact of gating them behind PNC. I am not sure if that is something that is wanted or planned.

**Comment by MuteTiefling** [comment]
There's already an issue about unifying latex.

I think the maybe a better idea world be to remove IF plastic entirely and have it use cured rubber instead.

So thermal latex would go away, replaced by IF latex. That latex would process the same, either through thermal or IF. But the output would be rubber and cured rubber instead of plastic.

This way PNC would be the only plastic and it would remain mid game. While IF and thermal keep their early game processes.

**Comment by TemperedFool** [comment]
I love this idea, but is it too Expert-y for base E6? I don't think so, but thought it would be good to make sure that's the general feeling before proceeding. I know this technically makes both IF and Thermal a little less complicated since they are interchangeable, which is why I think it's fine personally, but it also takes them away from "Vanilla" IF and Thermal a bit.

**Comment by MuteTiefling** [comment]
Fair points, and realistically...not all plastics are equal. HDPE is another plastic, yet rarely brought up in this type of discussion. Probably due to the difference in naming. 

HDPE is what we use for pipes and such. PNC Plastic is probably more similar to Lego (ABS). And IF's Plastic is probably more similar to PVC given the way it's used.

They aren't necessarily interchangeable.  

So leaving the plastic side of things as is could be fine. And the latex unification wouldn't actually take anything away from IF or Thermal. Both methods would co-exist, just the outputs would change.

**Comment by aqpoyo** [comment]
I also think it is good as it is otherwise with a simple setup as the IF one, you wouldn't be forced to the PNC one and the drones would be way cheaper. I don't like when in the modpacks they use same recipes for different mods, like it is now you learn new mods that probably you wouldn't use because you would use the easiest way always.
